modules/runner/src/integrationTest/java/org/apache/ignite/internal/schemasync/ItSchemaSyncSingleNodeTest.java

```diff
@@ -121,7 +121,7 @@ void readWriteOperationInTxAfterAlteringSchemaOnTargetTableIsRejected(Operation
 
     private void createTable() {
         cluster.doInSession(0, session -> {
-            executeUpdate("CREATE TABLE " + TABLE_NAME + " (id int PRIMARY KEY, val varchar)", session);
+            executeUpdate("CREATE TABLE " + TABLE_NAME + " (id int, val varchar, PRIMARY KEY USING HASH (id))", session);
```

Comment:
1. This test does not care about index type.
2. An error in OrderedPublisher is not correctly propagated (https://issues.apache.org/jira/browse/IGNITE-21684). So error message checks in this test are going to fail, if a sorted index is used.